opencodeblocks/scene/from_ipynb_conversion.py

```diff
@@ -0,0 +1,183 @@
+""" Module for converting ipynb data to ipyg data """
+
+from typing import OrderedDict, List
+
+from opencodeblocks.scene.ipynb_conversion_constants import *
+
+
+def ipynb_to_ipyg(data: OrderedDict) -> OrderedDict:
+    """Convert ipynb data (ipynb file, as ordered dict) into ipyg data (ipyg, as ordered dict)"""
+
+    blocks_data: List[OrderedDict] = get_blocks_data(data)
+    edges_data: List[OrderedDict] = get_sockets_data(blocks_data)
```

Comment:
I still think this should be edges_data though
```suggestion
    edges_data: List[OrderedDict] = get_edges_data(blocks_data)
```